mmdet3d/core/visualizer/show_result.py

```diff
@@ -89,6 +100,7 @@ def show_result(points, gt_bboxes, pred_bboxes, out_dir, filename):
         _write_ply(points, osp.join(result_path, f'{filename}_points.obj'))
 
     if pred_bboxes is not None:
+        pred_bboxes[..., 2] += pred_bboxes[..., 5] / 2
         pred_bboxes[:, 6] *= -1
```

Comment:
These two lines seem to be closely related to the input bbox type? If it does hack something, maybe need some comments or minor adjustments